pandas/io/tests/parser/common.py

```diff
@@ -836,7 +836,8 @@ def test_integer_overflow_bug(self):
         result = self.read_csv(StringIO(data), header=None, sep=' ')
         self.assertTrue(result[0].dtype == np.float64)
 
-        result = self.read_csv(StringIO(data), header=None, sep='\s+')
+        result = self.read_csv(StringIO(data), header=None,
```

Comment:
why do you need to change these tests? this is translated to the c-engine with delim whitespace=True so it shouldn't hit your change